src/main/resources/i18n/strings.properties

```diff
@@ -388,6 +388,10 @@ passwordStrength.messageLabel.2=Fair
 passwordStrength.messageLabel.3=Strong
 passwordStrength.messageLabel.4=Very strong
 
+#Recovery Key Password Reset Success
+passwordReset.success.message=Password reset done
+passwordReset.success.description=Your password has been successfully reset.
```

Comment:
The description should not be a mere repetetion of the message, but give the user more info about the matter. Like `You can unlock your vault with the new password` or something similar.